cluster/images/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.17.1
+FROM alpine:3.18
```

Comment:
I'd prefer the 3.18.2 which is available now:
https://hub.docker.com/layers/library/alpine/3.18.2/images/sha256-25fad2a32ad1f6f510e528448ae1ec69a28ef81916a004d3629874104f8a7f70?context=explore

Reply:
good

Reply:
Rebased.